tinc/scripts/setup-node.sh

```diff
@@ -69,6 +81,11 @@ if [ -n "$HOSTNAME" ]; then
    echo "Using hostname: $HOSTNAME"
 fi
 
+# Regenerate the certificate
+if $application_enabled; then
```

Comment:
where this $application_enabled variable is declared?
I don't see this anywhere else in this script